src/store/reducers/tenantOverview/topNodesByLoad/utils.ts

```diff
@@ -0,0 +1,20 @@
+import type {TComputeInfo} from '../../../../types/api/compute';
+import {generateEvaluator} from '../../../../utils/generateEvaluator';
+import {prepareComputeNodes} from '../../nodes/utils';
+import type {TopNodesByLoadHandledResponse} from './types';
+
+export const prepareTopComputeNodesData = (data: TComputeInfo): TopNodesByLoadHandledResponse => {
+    const preparedNodes = prepareComputeNodes(data.Nodes, data.Tenants);
+
+    let sortedNodes = preparedNodes;
+
+    if (!data.Nodes) {
+        sortedNodes = preparedNodes.sort((a, b) => Number(b.LoadAverage) - Number(a.LoadAverage));
+    }
+
+    return {
+        Nodes: sortedNodes,
+    };
+};
```

Comment:
1) `LoadAverage` is an array
2) You don't need `/compute`
3) Old versions of `/nodes` could be not filtered on backend properly. So you should process the result (filter it and limit to 5)